src/lib/ai-mcp.test.ts

```
@@ -0,0 +1,157 @@
+// @ts-nocheck - AI SDK v5 beta has type issues that need to be resolved
+import test from 'ava'
+import './ai.js' // Import AI functionality
+import { tool } from 'ai'
+import { config } from 'dotenv'
+import { resolve } from 'path'
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)